ci/test_wheel_cudf.sh

```diff
@@ -3,8 +3,20 @@
 
 set -eou pipefail
 
+# Set the manylinux version to pull the newer ABI wheels when testing on Ubuntu
+# 20.04 or later. Note that this logic is specific to the current set of
+# citestwheel images and would need to change if the images change.
+manylinux="manylinux_2_17"
+if command -v lsb_release >/dev/null 2>&1 ; then
```

Comment:
As before, I think we can derive this programmatically by parsing output of `ldd --version` or some other means.